src/utils.ts

```diff
@@ -1,32 +1,57 @@
-import { Periods } from "./types";
+import * as fs from "fs";
+import { Periods, Levels } from "./types";
 
 const periods: Periods = {
   w: 604800,
   d: 86400,
   h: 3600,
 };
 
+const levels: Levels = {
+  M: 0,
+  m: 1,
+  p: 2,
+};
+
 export const isValidDate = (date: string): boolean => {
   //const regex = new RegExp(/^\d{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])$/)
 
   //return regex.test(date)
-  const [year] = date.split('-')
-  return year.length === 4
-}
+  const [year] = date.split("-");
+  return year.length === 4;
+};
 
 export const isValidWarnOption = (option: string): string | boolean => {
   return option.match(/^\d+[w|d|h]$/) ? option : false;
 };
 
+export const isValidWarnWhenOption = (option: string): string | boolean => {
+  return option.match(/^\d+[M|m|p]$/) ? option : false;
```

Comment:
I think it will be a bit more clear if this function just returns a boolean. What do you think?